client/src/featureform/cli.py

```diff
@@ -187,4 +186,4 @@ def apply(host, cert, insecure, local, files, dry_run):
 
 
 if __name__ == '__main__':
-    cli()
+    cli()
```

Comment:
Whats going on here?

Reply:
I removed the changes to this file so it's the same as in the branch I'm merging into, this is just a weird spurious diff.